flycheck-coala.el

```diff
@@ -63,7 +63,7 @@
 (flycheck-define-checker coala
   "A checker using coala.
 
-See URL `http://coala-analyzer.org/'."
+See URL `http://coala.io'."
```

Comment:
Not sure if https is required since we're not providing any identifying information on that page.
The donate link is redirected, and does have a secure url.